.all-contributorsrc

```diff
@@ -1980,6 +1980,15 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "kefranabg",
+      "name": "Franck Abgrall",
+      "avatar_url": "",
```

Comment:
TBD feel free to update these as you see fit

Reply:
@kefranabg feel free to cherry pick the commits over !

Reply:
Hi @liyiy I just thought about the fact that I won't be able to push cherry picked commits here as I don't have write access to this repository. Anyway I don't think it will be possible to cherry pick as my commits comes from a fork (different repos) 
 😢 
 
 Another option would to expetionally bypass the status check validation to merge [this PR](https://github.com/PostHog/posthog/pull/10621#issuecomment-1203233625). Admins can do that.
 
 Unless the failing GH workflow is really important to check before the PR get merged?

Reply:
oh :( I'll see if I can get an admin to help me, do you mind fixing the merge conflict for that PR first then?

Reply:
@li, @kefranabg can't, but you can cherry pick his commits into this PR :)

Reply:
@liyiy Done, I just fixed the conflicts